src/sandbox/functionsConsumer/index.ts

```diff
@@ -0,0 +1,246 @@
+import { Location } from "@chainlink/functions-toolkit/dist/types";
```

Comment:
Not sure I understand what the purpose of this file is?  We should call out that the FunctionsConsumer is an example contract, not a requirement?

Reply:
Yes, FunctionsConsumer is the example contract, I'll try to state it clear in the documentation.